tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaApisTest.java

```diff
@@ -319,6 +322,43 @@ public void testConsumerListOffsetLatest() throws Exception {
         assertEquals(listOffsetResponse.responseData().get(tp).timestamp, Long.valueOf(0));
     }
 
+    @Test(timeOut = 60000)
+    public void testFetchMaxBytes() throws Exception {
+        String topicName = "testMaxBytesTopic";
+        int maxBytes = 1000;
+        int maxPartitionBytes = 500;
+
+        // create partitioned topic.
+        admin.topics().createPartitionedTopic(topicName, 2);
+        List<TopicPartition> topicPartitions = new ArrayList<>();
+        TopicPartition tp1 = new TopicPartition(topicName, 0);
+        TopicPartition tp2 = new TopicPartition(topicName, 1);
+        topicPartitions.add(tp1);
+        topicPartitions.add(tp2);
+
+        // producing data and then consuming.
+        KafkaProducer<String, String> kProducer = createKafkaProducer();
+        produceData(kProducer, topicPartitions, 20);
+        KafkaConsumer<String, String> consumer = createKafkaConsumer(5000, 1, maxBytes, maxPartitionBytes);
```

Comment:
I changed this line to

```java
KafkaConsumer<String, String> consumer = createKafkaConsumer(5000, 1);
```

and run this test in my local env. The test still passed. It might be related to `maxReadEntriesNum` config. Please add a contrast test to prove that with default max bytes, the fetched bytes will be larger than the `maxBytes` and `maxPartitionBytes`.

Reply:
Because the maxBytes and maxPartitionBytes settings are too large, causing maxReadEntriesNum to take effect, so
I reduced maxBytes and maxPartitionBytes to achieve the desired effect： @BewareMyPower 

        int maxBytes = 500;
        int maxPartitionBytes = 100;